hr_employee_relative/README.rst

```diff
@@ -0,0 +1,79 @@
+=====================
```

Comment:
@pedrobaeza the README file is auto-generated when merging. Should it be included in the migration and adapted or just removed so that later it can be added automatically?

Reply:
Don't touch it to reduce the diff.